app/views/jobs/getjob.html.erb

```diff
@@ -1,55 +1,167 @@
 <% content_for :javascripts do %>
   <%= javascript_include_tag "sorttable" %>
 <% end %>
-
-<h1>Job <%= @job[:name]%></h1>
 <p>
 <%= link_to "Return to Jobs", {:action=>"index"} %>
 </p>
-<h2>Summary</h2>
-<ul>
-<li>Job ID: <%= @job[:id] %>
-<li>State: <%= @job[:state] %>
-<li>Status: <%= @job[:status] %>
-<li>Elapsed time: <%= @job[:elapsed] %> secs
-<li>VM instance: <%= @job[:rjob]["vm"]["id"] %>
-</ul>
+<h3>Job <%= @job[:id] %> Summary</h3>
+<div class="jobs-table-div">
+  <table class="jobs-table">
+    <tbody>
+      <tr>
+        <td class="left">
+          Name
+        </td>
+        <td class="right">
+          <%= @job[:name] %>
+        </td>
+      </tr>
+      <tr>
+        <td class="left">
+          ID
+        </td>
+        <td class="right">
+          <%= @job[:id] %>
+        </td>
+      </tr>
+      <tr>
+        <td class="left">
+          State
+        </td>
+        <td class="right">
+          <%= @job[:state] %>
+        </td>
+      </tr>
+      <tr>
+        <td class="left">
+          Status
+        </td>
+        <td class="right">
+          <%= @job[:status] %>
+        </td>
+      </tr>
+      <tr>
+        <td class="left">
+          Elapsed time
+        </td>
+        <td class="right">
+          <%= @job[:elapsed] %> secs
+        </td>
+      </tr>
+      <tr>
+        <td class="left">
+          VM instance
+        </td>
+        <td class="right">
+          <%= @job[:rjob]["vm"]["id"] %>
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
 
-<h2>Job Parameters</h2>
-<ul>
-<li>VM image: <%= @job[:rjob]["vm"]["name"]%>
-<li>Job Timeout: <%= @job[:rjob]["timeout"] %> secs
```

Comment:
It seems that this information isn't displayed in the new layout?

Reply:
You're right, it should be there now